File: Challenge21.js
function string_match(a, b){
 var count = 0; var asub = ''; var bsub = '';
  for (var i = 0; i < a.length-1; i++){
    asub = a.slice(i, i+2);
    bsub = b.slice(i, i+2);
    if (asub == bsub){
      count = count +1
    }
}return count}

File: Challenge21.py
def string_match(a, b):
  count = 0
  asub = ''
  bsub = ''
  for i in range(len(a)-1):
    asub = a[i: i+2]
    bsub = b[i: i+2]
    if asub == bsub:
       count = count +1
  return count
